apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/EventActivity.java

```diff
@@ -103,6 +110,41 @@ public void onClick(View v) {
                 startActivity(intent);
             }
         });
+
+        /* Init pause/resume buttons. */
+        /* TODO remove reflection once SDKs are released. */
+        mPauseTransmissionButton = findViewById(R.id.pause_transmission_button);
+        mPauseTransmissionButton.setOnClickListener(new View.OnClickListener() {
+
+            @Override
+            public void onClick(View v) {
+                AnalyticsTransmissionTarget target = getSelectedTarget();
+                try {
+                    final Method pauseMethod = target.getClass().getMethod("pause");
+
+                    @SuppressWarnings("unchecked")
+                    AppCenterFuture<Void> future = (AppCenterFuture<Void>) pauseMethod.invoke(null);
+                    future.get();
+                } catch (Exception e) {
+                    e.printStackTrace();
```

Comment:
Please align reflection code with collectdeviceId, also we do not log using e.printStackTrace as it does use the System log tag instead of app log tag (and some devices disable it).